app/javascript/src/components/actions/TeamsActions.js

```diff
@@ -5,7 +5,11 @@ import type {
   Actopm$SetCurrentTeam
 } from "flow-typed";
 import type { Dispatch } from "redux-thunk";
-import { getTeams, changeCurrentTeam } from "../../services/api/teams_api";
+import {
+  getTeams,
+  changeCurrentTeam,
+  fetchCurrentTeam
```

Comment:
I'm just curious - should we enforce some sort of common nomenclature here? We're using `getTeams` - so, "get" - to access an array of entries, and now `fetchCurrentTeam()` to get a single element. I'm okay with using "fetch", but we should then try to stick to this for future actions. Other option would be to also use "get" here.